src/cmds/scripts/pbs_migration_assistant.bat

```diff
@@ -15,28 +15,15 @@ echo off
 SETX /m PBS_CONF_FILE "%WINDIR%\Temp\PBS Pro Backup\pbs.conf"
 echo In Backup Mode...
 echo ### Backing up Files...
-%backupcmd% "%pbs_exec_dir%\lib\liblmx-altair.dll" "%backupdir%MyDLLBackUp"
```

Comment:
Thanks for the update. On second thoughts, I agree with earlier change of removing this file altogether...:-) The file was originally designed to automate the migration upgrade steps on Windows platform but in it's current form, it is a dummy.